### PR TITLE
Add .clang-format and re-format worker source

### PR DESCRIPTION
**Pull request by KeyMaster-**
Formatted with clang-format version 10.0.0 on windows.
I ran `clang-format -i <file>` on both worker's startup.cc files, which are the only C++ source files in this project.

I've copied the .clang-format file from https://github.com/spatialos/CExampleProject/blob/master/.clang-format, with one change of making the language version be c++14

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+ContinuationIndentWidth: 4
+Cpp11BracedListStyle: true
+DerivePointerAlignment: false
+DisableFormat: false
+ExperimentalAutoDetectBinPacking: false
+FixNamespaceComments: true
+ForEachMacros:
+  - foreach
+  - Q_FOREACH
+  - BOOST_FOREACH
```
Comment (simonsarginson) [inline .clang-format:60]: ?

File: .clang-format [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+SpaceBeforeAssignmentOperators: true
+SpaceBeforeParens: ControlStatements
+SpaceInEmptyParentheses: false
+SpacesBeforeTrailingComments: 2
+SpacesInAngles: false
+SpacesInContainerLiterals: true
+SpacesInCStyleCastParentheses: false
+SpacesInParentheses: false
+SpacesInSquareBrackets: false
+Standard: Cpp11
```
Comment (simonsarginson) [inline .clang-format:108]:
```suggestion
Standard: Cpp14
```
?

File: .clang-format (hunk 1 above)
Comment (simonsarginson) [inline .clang-format:60]: ?

File: .clang-format (hunk 2 above)
Comment (KeyMaster-) [reply .clang-format:108]: ? indeed 🤔 I just copied this from the https://github.com/spatialos/CExampleProject to be consistent across example projects, not sure how that formatting file came to be.

File: .clang-format (hunk 1 above)
Comment (KeyMaster-) [reply .clang-format:60]: ? indeed 🤔 I just copied this from the https://github.com/spatialos/CExampleProject to be consistent across example projects, not sure how that formatting file came to be.

File: .clang-format (hunk 2 above)
Comment (KeyMaster-) [reply .clang-format:108]: Good catch, might even be worth bumping further - will need someone from worker to confirm.

Comment (simonsarginson) [reply .clang-format:108]: 14 is the current limit on consoles iirc.

**Review by grandseiken — APPROVED** [review]

File: .clang-format (hunk 2 above)
Comment (KeyMaster-) [reply .clang-format]: Bumped it to c++14, it makes sense as a default. Not that `Cpp11` is actually a deprecated alias to mean "latest language standard" in clang-format, though I don't suspect that's what was intended here :D

**Pull request merged**